hypervisor/include/dm/vpci.h

```diff
@@ -184,4 +184,28 @@ int32_t vpci_assign_pcidev(struct acrn_vm *tgt_vm, struct acrn_assign_pcidev *pc
 int32_t vpci_deassign_pcidev(struct acrn_vm *tgt_vm, struct acrn_assign_pcidev *pcidev);
 struct pci_vdev *vpci_init_vdev(struct acrn_vpci *vpci, struct acrn_vm_pci_dev_config *dev_config, struct pci_vdev *parent_pf_vdev);
 
+static inline bool is_pci_io_bar(struct pci_vbar *vbar)
+{
+        return ((vbar->bar_type.io_space.indicator == 1U) && (!vbar->is_mem64hi));
+}
+
+static inline bool is_pci_mem_bar(struct pci_vbar *vbar)
+{
+        return ((vbar->is_mem64hi) || ((vbar->bar_type.mem_space.indicator == 0U)));
+}
+
+static inline bool is_pci_reserved_bar(struct pci_vbar *vbar)
+{
+        return ((vbar->bar_type.mem_space.indicator == 0U) && ((vbar->bar_type.mem_space.mem_type & 0x1U) == 0x1U) && (!vbar->is_mem64hi));
+}
+
+static inline bool is_pci_mem32_bar(struct pci_vbar *vbar)
+{
+        return ((vbar->bar_type.mem_space.indicator == 0U) && (vbar->bar_type.mem_space.mem_type == 0U) && (!vbar->is_mem64hi));
+}
+
+static inline bool is_pci_mem64_bar(struct pci_vbar *vbar)
```

Comment:
Per your code, this is is_pci_mem64lo_bar

Reply:
Yes, will change to is_pci_mem64lo_bar()